eng/ApiListing.targets

```diff
@@ -30,6 +18,24 @@
     <MakeDir Directories="$(_RefSourceOutputPath)"/>
   </Target>
 
+  <Target Name="_SetExportApiInnerTarget">
+    <PropertyGroup>
+      <InnerTargets>ExportApi</InnerTargets>
+    </PropertyGroup>
+  </Target>
+
+  <Target 
+    Name="ExportApiCrossTarget"
```

Comment:
Do you have to do the cross targeting explicitly? I wouldn't expect that to be necessary.

Reply:
Yeah, we need to be running in the context of a TargetFramework but the script invokes `dotnet build` without passing a TFM

Reply:
Wouldn't it be simple enough to condition the target on TargetFramework being set? For cross builds TargetFrameworks is generally set but for inner builds TargetFramework is set. Or maybe even use IsCrossTargetBuild, see https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.targets#L15.

Reply:
I'm already using the `IsInnerBuild `. See https://github.com/Azure/azure-sdk-for-net/pull/20701/files#diff-51ed978cc2232baec9f971caf6f9d50a4bc3b3cbd2834ec130d5ce4234a57978R30 

But someone needs to invoke the ExportApi for every target framework as it doesn't happen automatically.

Reply:
Ok I think I'm following the logic now. We could probably eliminate `_SetExportApiInnerTarget` build and use the inverse condition on IsCrossTargetingBuild to draw to ensure the 2 targets are mutually exclusive but what you have now also works as it stands.